### PR TITLE
practical: add a tutorial directive

### PR DESCRIPTION
**Pull request by oliver-sanders**
Add a directive which generates a box with a link to a tutorial in it.

We might want to improve the formatting at some point, be centralising this into a directive we can easily tweak things down the road.

> **@MetRonnie:** Pinged you on this as it's a good simple introduction to the documentation infrastructure and Sphinx plugins. This is the cylc-sphinx-extensions repo, these extensions are used in both the Cylc and Rose documentation. It's auto-documenting, run `make clean html`.

### CODE REVIEW

**Comment by oliver-sanders** [comment]
> conda (rather than pip) install pygraphviz

Yep, `pygraphviz` is just the Python bindings to `graphviz` which don't get installed via pip (same sort of deal with `pygtk` and `gtk`, however, not the same with `pyzmq` and `zmq`). Either way, when it's not python definitely better to go via Conda.

**Comment by MetRonnie** [comment]
Sorry, what is meant to happen when you run `make clean html`? I just end up with a bunch of `.rst`s in the `extensions/` dir, e.g. `cylc.sphinx_ext.practical.rst`
```rst
practical
=========

.. automodule:: cylc.sphinx_ext.practical
```
And that's all there is in the file

**Comment by oliver-sanders** [comment]
That's a build error, what `sphinx-build` version do you have installed.

**Comment by MetRonnie** [comment]
Ah, always with the `pip install -e .[all]`!

**Comment by MetRonnie** [comment]
So if this provides a link to the tutorial rather than the tutorial itself, might it be better to name the title of the admonition "Link to tutorial", or the content "There is a tutorial for this at: <link>"?

**Comment by oliver-sanders** [comment]
Added an extra commit onto the end, [this comment](https://github.com/cylc/cylc-doc/pull/148#issuecomment-676533324) explains what it does.

**Review by MetRonnie — APPROVED** [review]

File: cylc/sphinx_ext/practical/_static/css/tutorial.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.admonition.tutorial-ref .admonition-title:before {
+    content: "";
+}
```
Comment (oliver-sanders) [inline cylc/sphinx_ext/practical/_static/css/tutorial.css:3]: This adds an info (letter `i` in a circle) to the admonition.

**Comment by oliver-sanders** [comment]
(changed notice to "Related Tutorial")

**Review by MetRonnie — APPROVED** [review]

**Comment by oliver-sanders** [comment]
ok, i'll push the merge button,

**Pull request merged**